### PR TITLE
Open internet feature

### PR DESCRIPTION
**Pull request by Unicus-Tom**
Feature is complete but some quality of life things have not been added. And there might be risk of database desync. 
I would like reviewers to start sepes and do practical tests as well in case I missed some edge cases.

### CODE REVIEW

File: FrontEnd/src/components/PodPage.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import PodRules from './PodRules';
 import PodDataset from './PodDataset.js';
 
 import * as StudyService from "../studyService"
 
 import Sepes from '../sepes.js';
+import Nsgswitch from './nsgSwitch';
 const sepes = new Sepes();
-
+var self = this;
```
Comment (gjersvik) [inline FrontEnd/src/components/PodPage.js]: This one is dangerous hope that self is not used. Do't even know what self referees to but nothing good.
The "let self = this" should only be used methods of classes. This line must be removed.

**Review by gjersvik — CHANGES_REQUESTED** [review]
Found a very dangerous js line. Remove.

File: FrontEnd/src/components/PodPage.js (hunk 1 above)
Comment (Unicus-Tom) [reply FrontEnd/src/components/PodPage.js]: It was a leftover of an attempted bug fix where a function/class did not get sent in the correct context to function correctly.
It was unused in the code

**Review by gjersvik — APPROVED** [review]
Looks god to me hope it works.

File: FrontEnd/src/components/SepesPodList.js [hunk 2]
```diff
@@ -12,16 +12,16 @@ export default function SepesPodList(props) {
...
                 <div key={item.podId} className="pod" onClick={() => props.openPod(item)}>
                     <p style={{ fontWeight: "bold" }}>{item.podName}</p>
-                    <p style={{ fontSize: 10 }}>
+                    <div style={{ fontSize: 10 }}>
                         Incoming
                     {item.incoming.map(rule => (
-                            <div>{rule.port} {rule.ip}</div>
-                        ))}</p>
-                    <p style={{ fontSize: 10 }}>
+                            <p>key={rule.port+rule.ip} {rule.port} {rule.ip}</p>
```
Comment (Kris2ffer) [inline FrontEnd/src/components/SepesPodList.js]: Shpuld be `<p key={rule.port+rule.ip}>{rule.port} {rule.ip}</p>`

**Review by Kris2ffer — CHANGES_REQUESTED** [review]

File: FrontEnd/src/components/SepesPodList.js (hunk 2 above)
Comment (Unicus-Tom) [reply FrontEnd/src/components/SepesPodList.js]: Fixed

**Review by Kris2ffer — APPROVED** [review]
